app/src/main/java/com/github/houseorganizer/houseorganizer/panels/login/RegisterEmail.java

```diff
@@ -62,11 +62,18 @@ protected void onCreate(Bundle savedInstanceState) {
         );
     }
 
-
+    private void initializeVariables() {
+        mAuth = FirebaseAuth.getInstance();
+        isEmailAlreadyUsed = false;
+        email = ((EditText) findViewById(R.id.reg_enter_email)).getText().toString();
+        password = ((EditText) findViewById(R.id.reg_enter_password)).getText().toString();
+        confPassword = ((EditText) findViewById(R.id.reg_confirm_password)).getText().toString();
```

Comment:
I I think the elements that are in the fields are supposed to be retrieved only when clicking the button, here you take them at the beginning when they are empty I think